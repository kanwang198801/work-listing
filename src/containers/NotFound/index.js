import React from 'react';
import Theme from '../../components/Theme';

const NotFound = () => {
  return (
    <Theme>
      <h1>The page is not found</h1>
    </Theme>
  );
};
export default NotFound;
