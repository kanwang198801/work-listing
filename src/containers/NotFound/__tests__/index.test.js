import React from 'react';
import renderer from 'react-test-renderer';
import { render } from '@testing-library/react';
import NotFound from '../index';

describe('NotFound Test Suite', () => {
  it('snapshot', () => {
    const tree = renderer.create(<NotFound />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders the Works component', () => {
    const component = render(<NotFound />);
    expect(component).toBeTruthy();
  });
});
