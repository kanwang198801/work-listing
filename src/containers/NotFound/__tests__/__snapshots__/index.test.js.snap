// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound Test Suite snapshot 1`] = `
<section
  className="ant-layout layout"
  style={
    Object {
      "minHeight": "100vh",
    }
  }
>
  <main
    className="ant-layout-content main"
  >
    <div>
      <h1>
        The page is not found
      </h1>
    </div>
  </main>
  <footer
    className="ant-layout-footer"
    style={
      Object {
        "textAlign": "center",
      }
    }
  >
    Copyright Â©
  </footer>
</section>
`;
